nncf/torch/__init__.py

```diff
@@ -50,6 +50,7 @@
 from nncf.torch.model_creation import is_wrapped_model
 from nncf.torch.model_creation import wrap_model
 from nncf.torch.model_creation import load_from_config
+from nncf.experimental.torch2.function_hook.serialization import get_config
```

Comment:
I would suggest to implement `get_config` function in the model_creation.py to support `NNCF_TORCH_LEGACY_TRACING` in case of the user migrate to `nncf.torch.get_config`. I hope this is not a big deal.

I would suggest to add a telemetry for get_config function.

Reply:
done